addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -86,8 +61,9 @@ export class MessageList extends Component {
                 threadViewer: threadView && threadView.threadViewer,
             };
         });
-        // useUpdate must be defined after useRenderedValues to guarantee proper
-        // call order
+        // useModels and useUpdate must be defined after useRenderedValues to
+        // guarantee proper call order
```

Comment:
Comment is too obscure. Briefly hint at why it guarantee proper call order.

Reply:
good idea